d2bs/kolbot/libs/bots/DiabloHelper.js

```diff
@@ -141,8 +191,15 @@ function DiabloHelper() {
 	};
 
 	this.infectorSeal = function () {
+		Precast.doPrecast(true);
```

Comment:
Maybe change it so, it only does this when it did the entrance?

Reply:
previously, the `Precast.doPrecast(true);` call was just in front of infector, so it was specifically to protect chars before they go to infector. I just moved it inside the infector seal call, so it was a little neater and could be reused easier